backend/backend.go

```diff
@@ -232,30 +232,34 @@ type Backend struct {
 	tstCheckAccountUsed func(accounts.Interface) bool
 	// For unit tests, called when `backend.maybeAddHiddenUnusedAccounts()` has run.
 	tstMaybeAddHiddenUnusedAccounts func()
+
+	restartingInTestnet bool
```

Comment:
This is redundant with `backend.config.AppConfig().Backend.RestartInTestnet`, so I would remove this here to improve clarity. Otherwise I am inclined  to think that this value is somehow special or different to what's in the config.

Reply:
of course! I completely missed that I could just use that! Thanks

Reply:
Right, my bad. For clarity, would this maybe help?

- rename it to `testing bool`
- set it to `backend.config.AppConfig().Backend.Testing || backend.arguments.Testing()` followed by modifying the config to set it back to `false`
- `func (backend *Backend) Testing() bool { return backend.testing }`

then the logic would be concentrated and easier to follow I think.